apps/testapp/src/components/RpcMethods/shortcut/walletTxShortcuts.ts

```diff
@@ -0,0 +1,22 @@
+import { ADDR_TO_FILL } from './const';
+import { ShortcutType } from './ShortcutType';
+
+const walletSendTransactionShortcuts: ShortcutType[] = [
+  {
+    key: 'wallet_sendTransaction',
+    data: {
+      chainId: '84532',
+      sender: ADDR_TO_FILL,
+      calls: [],
+      capabilities: {
+        paymaster: {
+          url: 'https://paymaster.base.org',
```

Comment:
should update to include version as specified above

Reply:
thanks! will update

Reply:
addressed in [6178c33](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/6178c33e20166ef8e819447e8ed74dbd1299dfb7)